views/layout.erb

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Titillium+Web:ital,wght@0,300;0,400;0,700;1,300;1,400;1,700&display=swap" rel="stylesheet">
+
+  <link rel="stylesheet" href="/css/normalize.css">
+  <link rel="stylesheet" href="/css/drop-in.css">
+  <link rel="stylesheet" href="/css/application.css">
+
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+  <title>Chitter!</title>
+</head>
+
+<body>
+  <header class="center">
+    <h1><a href="/">Chitter!</a></h1>
+  </header>
+
+  <main>
+    <div class="topnav">
+      <a href="<%='/chitter/index'%>">Home</a>
+      <%if @user%>
+      <a href="#">Tagged Peeps</a>
+      <%# <a href="#">Link</a> %>
+      <form action="/sessions/destroy" method="post" style="float:right;">
```

Comment:
as you have css sheets set up you could move the styling of this table to the relevant stylesheet